### ISSUE TITLE
Trying to add this dependency to my package.json with latest commit. When installing, the external SpotifySDK is not installed along with the rest of the package. Why?

### ISSUE THREAD

**Issue by EmpeeDubs** [cjam/react-native-spotify-remote#214]
When including this dependency in my package.json file as such:

"react-native-spotify-remote": "git+https://github.com/cjam/react-native-spotify-remote.git#5e810b7f73f85c3204131c3ba0391b6a887b7344"

I do not get the external SDKs along with it. Normally I would install these manually, but I'm using Expo building and the dependency "URL" or location needs to include all of the files. Is this a bug? Or am I missing somthing?

If I install with just the package and the released version number, it all works fine, but my build needs the latest commit that's not released yet... any ideas?

**Comment by cjam** [comment]
yea, that doesn't surprise me given the nature of referencing a package by git commit (since there are git submodules that it wouldn't know to initialize).

That being said, have you tried the latest pre-release version?  It is accessible via the `@next`
https://www.npmjs.com/package/react-native-spotify-remote?activeTab=versions

Not very well advertised via npm, but something to try.

**Comment by EmpeeDubs** [comment]
Yeah, I tried the pre-release version, unfortunately it doesn't contain the commit from suryababus committed on Jul 14. I'll wait for a release with this commit. In the meantime, I went ahead and forked the latest commit and added the SDKs fully without the internal reference. 

I was just wondering if I was missing something about the functionality of npm or git and there was any easy way to deal with this. Thanks for replying!
